### PR TITLE
Countdown to the New Year: 31 Days of Ember Addons - ember-shepherd (Dec 20th 2019)

### LINKED ISSUE #403: Create a blog post for December 20, 2019 about the `ember-shepherd` addon
## Goal
Write a short blog post for the December blog series that is similar to the [example post]( https://github.com/ember-learn/ember-blog/blob/be8f29ba0ae8b5367d1758b1a150c1b503812a77/source/2019-12-01-countdown-to-the-new-year-ember-sortable.md).

## Criteria

-  [ ] file name should 2019-12-DD-countdown-to-the-new-year-ADDON-NAME.md where DD is replaced with the day and ADDON-NAME is replaced with the hyphenated name of the addon
- [ ] The opening sentence should be updated where COUNT is the sequential ordinal word. 

```
<b>This is the COUNT in our DecEmber series- <i>"Countdown to The New Year: 31 Days of 
Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and 
we hope you'll join us for the fun!</b>
```

- [ ] The first h2 should be "Day NUM" where NUM is the same as the DD in the date.
- [x] the link to the addon should point to the page on Ember Observer
- [x] There should be two sub sections: (both h3)- "What it does" and "Why I like it" 
- [ ] Include whether or not the addon has been updated yet for Ember Octane- if not, invite contributors to help get the addon Octane ready.
- [x] The closing sentence can be slightly altered but should feel similar: 
"Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy!"

### PR DESCRIPTION
**Pull request by RobbieTheWagner**
## What it does

When merged, this will create a new blog post for the DecEmber 2019 blog series.

Should be merged on December 20th, 2019.

Closes https://github.com/ember-learn/ember-blog/issues/403

### CODE REVIEW

File: source/2019-12-20-countdown-to-the-new-year-ember-shepherd.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+It allows you to show new users around your app, introduce new features to existing users, or suggest a flow your users might follow to optimally use your app.
+
+It is configurable to show modal overlays or not, and can be hooked up to your existing application flow to conditionally show steps or be controlled manually.
+
+### Why I Like It
+
+Tours are an often overlooked feature of apps, and can be crucial to providing a good user experience. Without them, a user may feel lost in your app or do things in a way you did not intend.
+
+ember-shepherd makes adding these useful tours to your Ember app much easier than hand rolling them yourself. With the provided Ember service, you can add steps and interact with your tour in a familiar Ember way.
```
Comment (MelSumner) [inline source/2019-12-20-countdown-to-the-new-year-ember-shepherd.md]:
```suggestion
The `ember-shepherd` addon makes adding these useful tours to your Ember app much easier than hand rolling them yourself. With the provided Ember service, you can add steps and interact with your tour in a familiar Ember way.
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for this :sparkles: writeup for the DecEmber series!

**Review by MelSumner — APPROVED** [review]
Thank you for writing this! :+1:

**Comment by RobbieTheWagner** [comment]
You're welcome!

**Pull request merged**